guides/common/modules/proc_synchronize-red-hat-insights-recommendations-for-hosts.adoc

```diff
@@ -0,0 +1,22 @@
+[id="synchronize-red-hat-insights-recommendations-for-hosts_{context}"]
+= Synchronize Red{nbsp}Hat Insights Recommendations for Hosts
```

Comment:
```suggestion
= Configuring Automatic Synchronization of Insights Recommendations for Hosts
```

Reply:
Got swapped between `ref` and `proc` while using the verb in gerundium. :smile: 
Thank you.